nilearn/plotting/html_document.py

```diff
@@ -105,9 +110,11 @@ def get_iframe(self, width=None, height=None):
         if height is None:
             height = self.height
         escaped = escape(self.html, quote=True)
-        wrapped = (f'<iframe srcdoc="{escaped}'
-                   f'width="{width}" height="{height}" '
-                   'frameBorder="0"></iframe>')
+        wrapped = (
+            f'<iframe srcdoc="{escaped}'
```

Comment:
i think that should be:
```suggestion
            f'<iframe srcdoc="{escaped}" '
```